mlops/parallelm/mlops/ml_metrics_stat/confusion_matrix.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+from parallelm.mlops.metrics_constants import ClassificationMetrics
+from parallelm.mlops.mlops_exception import MLOpsException
+from parallelm.mlops.stats.table import Table
+
+
+class ConfusionMatrix(object):
+    """
+    Responsibility of this class is basically creating table object out of input array and list of labels.
+    """
+
+    @staticmethod
+    def _get_mlops_cm_table_object(cm_nd_array, **kwargs):
```

Comment:
do we need to keep this a private method?

Reply:
i do not want it to be used without knowing what it does. User outside mlops should not use this at all. Thus.